src/include/souffle/profile/ProfileEvent.h

```diff
@@ -51,7 +51,7 @@ class ProfileEventSingleton {
 public:
     ~ProfileEventSingleton() {
         stopTimer();
```

Comment:
Do we need to do the same for stopTimer?

Reply:
Not as far as I can tell - the call to `stopTimer` doesn't use the `instance` method, which I believe is what's causing the issue. Plus, it stopped segfaulting with this change as is.